db/00-create-tables.sql

```diff
@@ -0,0 +1,28 @@
+create table if not exists users(
```

Comment:
What is the numeric file naming for?

Reply:
PG executes init scripts by alfabetical order. We need to make the tables first and then add the data. This the numbers. This is a pretty standard approach.

Reply:
Ahh gotcha, so do these get run by the postgres docker image on boot?

Reply:
Correct, thus the volume on the compose file